artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -934,6 +934,26 @@
                               a broker plugin
                            </xsd:documentation>
                         </xsd:annotation>
+                        <xsd:sequence>
```

Comment:
Is it worth instead of re-defineing what a property type is, re-naming and re-using the transformerProperty complex type which this is identical to?

also could the same file  /artemis-tools/src/test/resources/artemis-configuration.xsd be updated also.

Reply:
Sure, I was originally hesitant to mess with your transformerProperty but happy to do so if it is ok with you.  I will make the changes you suggested next week,

Thanks.